tests/protractor/e2e/registration-by-sms.js

```diff
@@ -0,0 +1,279 @@
+const utils = require('../utils');
+
+describe('registration transition', () => {
+
+  'use strict';
+
+  const PHONE = '+64271234567';
+  const FORM_NAME = 'Registration';
+
+  // contacts
+  const BOB_PLACE = {
+    _id: 'bob-contact',
+    reported_date: 1,
+    type: 'clinic',
+    name: 'Bob Place'
+  };
+  const CAROL = {
+    _id: 'carol-contact',
+    reported_date: 1,
+    type: 'person',
+    phone: PHONE,
+    name: 'Carol Carolina',
+    parent: BOB_PLACE,
+    patient_id: '05946',
+    sex: 'f',
+    date_of_birth: 1462333250374
+  };
+
+  const DOCS = [ BOB_PLACE, CAROL ];
+  const CONFIG = {
```

Comment:
A thought: at what point should this become an external JSON file we load a as a resource? I imagine with future registration tests we may want a generic cross-test config file?